compiler/pipes/calc-rl.cpp

```diff
@@ -115,6 +115,12 @@ void rl_common_calc(VertexPtr root, RLValueType expected_rl_type) {
       break;
     case op_ffi_new:
       break;
+    case op_yield:
+      kphp_error(false, "yield isn't supported");
+      break;
+    case op_yield_from:
+      kphp_error(false, "yield from isn't supported");
+      break;
```

Comment:
Is this code reachable?
I suppose it isn't true.

Reply:
You're right. I'll delete it.